conf/application.conf

```diff
@@ -96,11 +96,10 @@ microservice {
        port = 8300
     }
 
-    update-email {
-       protocol = http
-       host = localhost
-       port = 9898
+    customs-email-frontend {
+      url = "http://localhost:9898/manage-email-cds/service/eu-subsidy-compliance-frontend"
```

Comment:
I think this should either live in the `urls` object, or have the protocol/host/port pattern like we do for esc

Reply:
earlier I had protocol host patter here and the Damein said it will not work , and I had to change it